### ISSUE TITLE
Fonts blurry on Retina Screens

### ISSUE THREAD

**Issue by olilarkin** [charlieroberts/interface.js#12]
so far i wasn't able to add HiDPI support. If you've got a clue how to do it that'd be great! thanks

**Comment by charlieroberts** [comment]
Hi Oli,

I can take a look into this (and have noticed this problem before); hopefully the fix will be simple. Just to be clear, do you mean that you tried to edit the library to enable hidpi and were unable to do it successfully? Or that you were unable to figure out a workaround to do it with the library as is?

- Charlie

**Comment by olilarkin** [comment]
yes i tried to edit the library around here https://github.com/charlieroberts/interface.js/blob/master/index.js#L277

following this: https://gist.github.com/joubertnel/870190

I was able to scale the context to 2x, but was unable to get it to render at the correct size
